SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKSimulatedPlayerDeployment.cpp

```diff
@@ -0,0 +1,530 @@
+	SpatialGDKCloudLauncherSettings->SetSimulatedPlayerDeploymentName(InText.ToString());
+}
+
+void SSpatialGDKSimulatedPlayerDeployment::OnNumberOfSimulatedPlayersCommited(uint32 NewValue)
+{
+	USpatialGDKEditorCloudLauncherSettings* SpatialGDKCloudLauncherSettings = GetMutableDefault<USpatialGDKEditorCloudLauncherSettings>();
+	SpatialGDKCloudLauncherSettings->SetNumberOfSimulatedPlayers(NewValue);
+}
+
+void SSpatialGDKSimulatedPlayerDeployment::OnSimulatedPlayerLaunchConfigPathPicked(const FString & PickedPath)
+{
+	USpatialGDKEditorCloudLauncherSettings* SpatialGDKCloudLauncherSettings = GetMutableDefault<USpatialGDKEditorCloudLauncherSettings>();
+	SpatialGDKCloudLauncherSettings->SetSimulatedPlayerLaunchConfigPath(PickedPath);
+}
+
+FReply SSpatialGDKSimulatedPlayerDeployment::OnLaunchClicked()
+{
+	USpatialGDKEditorCloudLauncherSettings* SpatialGDKCloudLauncherSettings = GetMutableDefault<USpatialGDKEditorCloudLauncherSettings>();
+
+	if (!SpatialGDKCloudLauncherSettings->IsDeploymentConfigurationValidWithCheck()) {
+		FNotificationInfo Info(FText::FromString(TEXT("Deployment configuration is not valid.")));
+		Info.bUseSuccessFailIcons = true;
+		Info.ExpireDuration = 3.0f;
+
+		TSharedPtr<SNotificationItem> NotificationItem = FSlateNotificationManager::Get().AddNotification(Info);
+		NotificationItem->SetCompletionState(SNotificationItem::CS_Fail);
+
+		return FReply::Handled();
+	}
+
+	if (TSharedPtr<FSpatialGDKEditor> SpatialGDKEditorSharedPtr = SpatialGDKEditorPtr.Pin()) {
+		FNotificationInfo Info(FText::FromString(TEXT("Starting simulated player deployment...")));
+		Info.bUseSuccessFailIcons = true;
+		Info.bFireAndForget = false;
+
+		TSharedPtr<SNotificationItem> NotificationItem = FSlateNotificationManager::Get().AddNotification(Info);
+
+		NotificationItem->SetCompletionState(SNotificationItem::CS_Pending);
+
+		SpatialGDKEditorSharedPtr->LaunchCloudDeployment(
+			FSimpleDelegate::CreateLambda([NotificationItem]() {
+				NotificationItem->SetText(FText::FromString(TEXT("Successfully initiated launching of the cloud deployment.")));
+				NotificationItem->SetCompletionState(SNotificationItem::CS_Success);
+			}),
+			FSimpleDelegate::CreateLambda([NotificationItem]() {
+				NotificationItem->SetText(FText::FromString(TEXT("Failed to launch the DeploymentLauncher script properly.")));
+				NotificationItem->SetCompletionState(SNotificationItem::CS_Fail);
+			})
+		);
+		return FReply::Handled();
+	}
+
+	FNotificationInfo Info(FText::FromString(TEXT("Couldn't launch the deployment.")));
+	Info.bUseSuccessFailIcons = true;
+	Info.ExpireDuration = 3.0f;
+
+	TSharedPtr<SNotificationItem> NotificationItem = FSlateNotificationManager::Get().AddNotification(Info);
+	NotificationItem->SetCompletionState(SNotificationItem::CS_Fail);
+
+	return FReply::Handled();
+}
+
+FReply SSpatialGDKSimulatedPlayerDeployment::OnRefreshClicked()
+{
+	// TODO: Invoke Platform SDK to refresh the deployment list
```

Comment:
List is a function of the deployment launcher script - might be good to mention that here instead of Platform SDK